packages/react-core/src/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -0,0 +1,119 @@
+---
+id: Label group
+section: components
+cssPrefix: pf-c-label
+propComponents: ['LabelGroup', 'Label']
+isBeta: true
```

Comment:
I am not sure why this is not showing as Beta in the preview link.

Reply:
Fixed - corrected from `isBeta` to `beta`